ts/Series/Treegraph/TreegraphNode.ts

```diff
@@ -0,0 +1,195 @@
+/* *
+ *
+ *  (c) 2010-2022 Pawel Lysy Grzegorz Blachlinski
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+import type TreegraphSeries from './TreegraphSeries.js';
+import type TreemapNodeNS from '../Treemap/TreemapNode.js';
+
+import TreegraphPoint from './TreegraphPoint.js';
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+const {
+    seriesTypes: {
+        treemap: {
+            prototype: {
+                NodeClass: TreemapNode
+            }
+        }
+    }
+} = SeriesRegistry;
+
+namespace TreegraphNode {
+    export interface NodeValuesObject extends TreemapNodeNS.NodeValuesObject {}
+    export class Node extends TreemapNode {
+        public mod: number = 0;
+        public thread?: Node;
+        public ancestor: Node = void 0 as any;
+        public shift: number = 0;
+        public change: number = 0;
+        public children: Array<Node> = [];
+        public x?: number;
+        public y?: number;
+        public preX: number = 0;
+        public point: TreegraphPoint = void 0 as any;
+        public column: number = void 0 as any;
```

Comment:
Number properties should get a default value instead of `void 0 as any`. Use `NaN` if it should fail by default.